pkg/ddevapp/config.go

```diff
@@ -178,6 +177,29 @@ func (c *Config) Config() error {
 	return nil
 }
 
+// Validate ensures the configuraton meets ddev's requirements.
+func (c *Config) Validate() error {
+	// validate docroot
+	fullPath := filepath.Join(c.AppRoot, c.Docroot)
+	if _, err := os.Stat(fullPath); os.IsNotExist(err) {
+		return fmt.Errorf("no directory could be found at %s. Please enter a valid docroot in your configuration", fullPath)
```

Comment:
My thinking was that it would just be a warning about a potential problem rather than actually prevent the site from starting, but I can accept your answer either way :)